assets/javascripts/kitten/karl/pages/modal-edit-project/index.js

```diff
@@ -0,0 +1,271 @@
+import { createGlobalStyle } from 'styled-components'
+import { Grid, GridCol } from '../../../components/layout/grid'
+import { Modal } from '../../../components/molecules/modal-next'
+import { Text } from '../../../components/atoms/typography/text'
+import { Title } from '../../../components/atoms/typography/title'
+import { InfiniteIconNext } from '../../../components/graphics/icons-next/infinite-icon-next'
+import { ChronoIconNext } from '../../../components/graphics/icons-next/chrono-icon-next'
+import { CheckedIcon } from '../../../components/graphics/icons/checked-icon'
+import { pxToRem } from '../../../helpers/utils/typography'
+import classNames from 'classnames'
+import { COLORS } from '../../..'
+import { mq } from '../../../constants/screen-config'
+
+const GlobalStyle = createGlobalStyle`
+  button {
+    border-width: ${pxToRem(2)};
+    border-style: solid;
+    border-radius: ${pxToRem(8)};
+    background-color: ${COLORS.background1};
+    border-color: ${COLORS.line1};
+    padding: ${pxToRem(20)};
+    min-height: ${pxToRem(395)};
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    @media ${mq.mobile} {
+      min-height: ${pxToRem(285)};
+    }
+
+    :hover,
+    :focus {
+      cursor: pointer;
+
+      .k-ModalEditProject--button {
+        border-color: ${COLORS.primary2};
+        background-color: ${COLORS.primary2};
+      }
+    }
+  }
+
+  .k-ModalEditProject--paragraph {
+    flex: 1;
+    margin-bottom: ${pxToRem(20)}!important;
+    margin-top: 0;
+  }
+
+  .k-ModalEditProject--blockText {
+    display: flex;
+    align-items: baseline;
+    margin-top: ${pxToRem(10)};
+    line-height: normal;
+  }
+`
+
+const ModalEditProject = () => {
+  return (
+    <>
+      <GlobalStyle />
+      <Modal
+        size="big"
+        trigger={null}
+        isOpen
+        hasCloseButton
+        className="k-ModalEditProject--modal"
+      >
+        {() => (
+          <>
+            <Modal.Block>
+              <Modal.Title>
+                <Title
+                  tag="h1"
+                  modifier="quaternary"
+                  className={classNames(
+                    'k-u-align-left@m-up',
+                    'k-u-align-center@s-down',
+                    'k-u-margin-top-quadrupleHalf',
+                    'k-u-margin-bottom-tripleHalf',
+                  )}
+                >
+                  Créer un nouveau projet
+                </Title>
+              </Modal.Title>
+            </Modal.Block>
+            <Modal.Block>
+              <Grid>
+                <GridCol col-s="6">
+                  <button
+                    type="button"
+                    href="#"
+                    className={classNames(
+                      'k-u-margin-right-noneHalf@m-up',
+                      'k-u-margin-bottom-double@s-down',  
+                    )}
+                  >
+                    <Modal.Paragraph className="k-ModalEditProject--paragraph">
```

Comment:
Je pense que `Paragraph` serait mieux : `Modal.Paragraph` est fait pour être utilisé comme enfant direct d'une `Modal`, pas comme descendant indirect, si on change le comportement ou les marges de `Modal.Paragraph` ça risque de tout cassr

```suggestion
                    <Paragraph className="k-ModalEditProject--paragraph">
```